app/controllers/reports/groups_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Reports
+  class GroupsController < ApplicationController
+    before_action :authorize_user
+
+    def show
+      render json: body, status: :ok
```

Comment:
Poderia ser talvez só 
```suggestion
      render json: expenses.to_json, status: :ok
```
?? ou nem?

Reply:
Se for assim, não retorna:
```
"expenses": { [ expense1, expense2 ] }
```
Ele retorna:
```
{ [ expense1, expense2 ] }
```
Mas talvez não seja necessário ter o `"expenses": {}` 🤔

Reply:
Se for assim, não retorna:
```
"expenses": { [ expense1, expense2 ] }
```
Ele retorna:
```
{ [ expense1, expense2 ] }
```
Mas talvez não seja necessário ter o `"expenses": {}` 🤔

Reply:
Acho que não é necessário